docs/ad/index.md

```diff
@@ -86,6 +113,11 @@ If you see the detector pending in "initialization" for longer than a day, aggre
 
 Anomaly grade is a number between 0 and 1 that indicates the level of severity of how anomalous a data point is. An anomaly grade of 0 represents “not an anomaly,” and a non-zero value represents the relative severity of the anomaly. The confidence score is an estimate of the probability that the reported anomaly grade matches the expected anomaly grade. Confidence increases as the model observes more data and learns the data behavior and trends. Note that confidence is distinct from model accuracy.
 
+If you set the category field, you see an additional **Heat map** chart. The heat map correlates results for anomalous entities.
```

Comment:
one feedback we recieved from uxdr is that the initial status of the result / feature plots is empty, which might be confusing for users, can we add a notes here that the plots will be empty till users select the anomaly entity?

Reply:
And for high cardinality detector, we only show anomaly and feature line chart for time period which has anomaly (anomaly_grade > 0).

Reply:
added